src/app/pages/virtualization/components/instance-wizard/select-image-dialog/select-image-dialog.component.ts

```diff
@@ -75,7 +75,7 @@ export class SelectImageDialogComponent implements OnInit {
 
   constructor(
     private ws: WebSocketService,
-    private dialogRef: MatDialogRef<CreateInstanceFormComponent>,
+    private dialogRef: MatDialogRef<InstanceWizardComponent>,
```

Comment:
Actually incorrect reference here. DialogRef refers to the component that was open, so SelectImageDialog in this case.